backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -94,23 +95,23 @@ public class BulkUploadResultsToFhir {
 
   private final Map<String, String> testResultToSnomedMap =
       Map.of(
-          "Positive".toLowerCase(), "260373001",
-          "Negative".toLowerCase(), "260415000",
+          "Positive".toLowerCase(), "10828004",
+          "Negative".toLowerCase(), "260385009",
           "Detected".toLowerCase(), "260373001",
           "Not Detected".toLowerCase(), "260415000",
           "Invalid Result".toLowerCase(), "455371000124106");
```

Comment:
could we use the constants for these SNOMEDs?
https://github.com/CDCgov/prime-simplereport/blob/5a0799cca650f7d9eb2f9bb02a9d29b4979e383f/backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java#L138-L138
https://github.com/CDCgov/prime-simplereport/blob/5a0799cca650f7d9eb2f9bb02a9d29b4979e383f/backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java#L489-L491